ui/components/app/home-notification/home-notification.stories.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import HomeNotification from './home-notification.component';
+
+export default {
+  title: 'UI/HomeNotification',
+  id: __filename,
+  component: HomeNotification,
+  parameters: {},
+  argTypes: {
+    checkboxText: {
+      control: 'text',
+    },
+    acceptText: {
+      control: 'text',
+    },
+    ignoreText: {
+      control: 'text',
+    },
+    descriptionText: {
+      control: 'text',
+    },
+    infoText: {
+      control: 'text',
+    },
+  },
+};
+
+const noop = () => {
+  // noop
+};
+
+export const DefaultStory = (args) => (
+  <HomeNotification {...args} onAccept={noop} onIgnore={noop} />
+);
```

Comment:
You can use actions here and they will show up in the actions tab when you click on the accept or ignore buttons

```suggestion
    onAccept: {
      action: 'onAccept',
    },
    onIgnore: {
      action: 'onIgnore',
    },
    },
  },
};

export const DefaultStory = (args) => (
  <HomeNotification {...args} />
);
```

<img width="1234" alt="Screen Shot 2021-11-25 at 2 16 23 PM" src="https://user-images.githubusercontent.com/8112138/143504110-74dc9f6e-fb16-4015-af63-9eba5c587c2b.png">